python/tests/test_topology.py

```diff
@@ -4990,18 +4990,35 @@ class TestTrim(unittest.TestCase):
     """
     Test the trimming functionality
     """
+    def add_mutations(self, tables, position, ancestral_state, derived_states):
+        "Create a site at the specified position and randomly assign mutations to nodes"
+        site = tables.sites.add_row(position, ancestral_state)
+        for state in derived_states:
+            tables.mutations.add_row(
+                site, np.random.randint(tables.nodes.num_rows), state)
```

Comment:
If you want multiple mutations, might as well use the jukes cantor function. This is adding randomness without setting a seed.

Reply:
Good point about the seed. I didn't notice the JC function

Reply:
Ah, but I want to place the sites *at a specific position*, which is why I rolled my own. Then I can check if they have been deleted appropriately. The tsutil.jukes_cantor doesn't allow you to specify positions.

Reply:
Well, don't bother with randomly choosing the nodes then, put them over the samples. Either that or set a seed on np.random (non deterministic tests are bad news)

Reply:
Yep, over samples is sensible. I agree about non-determinism.